py-polars/tests/unit/io/test_scan.py

```diff
@@ -70,13 +95,123 @@ def data_file_single(session_tmp_dir: Path, data_file_extension: str) -> _DataFi
     return _DataFile(path=file_path, df=df)
 
 
+@pytest.fixture(scope="session")
+def data_file_glob(session_tmp_dir: Path, data_file_extension: str) -> _DataFile:
+    row_counts = [
+        100,
+        186,
+        95,
+        185,
+        90,
+        84,
+        115,
+        81,
+        87,
+        217,
+        126,
+        85,
+        98,
+        122,
+        129,
+        122,
+        1089,
+        82,
+        234,
+        86,
+        93,
+        90,
+        91,
+        263,
+        87,
+        126,
+        86,
+        161,
+        191,
+        1368,
+        403,
+        192,
+        102,
+        98,
+        115,
+        81,
+        111,
+        305,
+        92,
+        534,
+        431,
+        150,
+        90,
+        128,
+        152,
+        118,
+        127,
+        124,
+        229,
+        368,
+        81,
+    ]
```

Comment:
We don't use this elsewhere, but you can specify `  # fmt: skip` after the `]` bracket if you want to manually format your list and prevent ruff from reformatting, so it doesn't take up so much screen space, as in:

```python
row_counts = [
    100, 186, 95, 185, 90, 84, 115, 81, 87, 217, 126, 85, 98, 122, 129, 122, 1089, 82,
    234, 86, 93, 90, 91, 263, 87, 126, 86, 161, 191, 1368, 403, 192, 102, 98, 115, 81,
    111, 305, 92, 534, 431, 150, 90, 128, 152, 118, 127, 124, 229, 368, 81,
]  # fmt: skip
```

Sometimes the strict list formatting requirements create (IMO) a lot of wasted space.

Reply:
Thanks.